hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java

```diff
@@ -0,0 +1,4 @@
+package ua.ithillel.expensetracker.dto;
+
+public class ExpenseCategoryDTO {
+}
```

Comment:
class members are missing, perhaps you've forgotten to commit the changes?

Reply:
Yes, I forgot to record the changes. Fixed it